.github/workflows/nightly-tests.yml

```diff
@@ -68,8 +68,6 @@ jobs:
       run: rustup toolchain install 1.77-x86_64-unknown-linux-gnu
     - name: Install nightly
       run: rustup toolchain install nightly-2024-08-01-x86_64-unknown-linux-gnu
-    - name: Install riscv target
-      run: rustup target add riscv32imac-unknown-none-elf --toolchain nightly-2024-08-01-x86_64-unknown-linux-gnu
```

Comment:
We never needed this in the first place, because we have always used `-Zbuild-std=core,alloc`. We might want it if we drop support for ASM path, so that `-Zbuild-std=core,alloc` won't be needed anymore.